File: source/back-end/src/main/java/be/pxl/backend/service/ISensorUsageService.java
package be.pxl.backend.service;

import java.util.List;

import be.pxl.backend.entity.SensorUsage;

public interface ISensorUsageService {
	public SensorUsage find(int id);
	
	public List<SensorUsage> all();
	
	public void persist(SensorUsage sensorUsage);
	
	public void delete(int id);
	
}

File: Source/frontend/frontend.Service/ISensorUsageService.cs
﻿using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface ISensorUsageService
    {
        List<SensorUsage> All();
        SensorUsage Find(int id);
        void Add(SensorUsage sensorUsage);
    }
}
